xep-0060.xml

```diff
@@ -4412,7 +4418,7 @@ And by opposing end them?
     id='subman3'>
   <pubsub xmlns='http://jabber.org/protocol/pubsub#owner'>
     <subscriptions node='princely_musings'>
-      <subscription jid='bard@shakespeare.lit' subscription='subscribed'/>
+      <subscription jid='polonius@denmark.lit' subscription='subscribed'/>
```

Comment:
NIT: I briefly remember discussion about using `example.net` instead of `*.lit` since technically .lit could become a TLD in the future. I'm not super concerned either way, especially since this is just modifying it to be consistent with the text which already used `.lit`.